pkg/git/lsremote.go

```diff
@@ -158,14 +163,14 @@ func (g *git) remoteSHAChanged(branch, sha string) (bool, error) {
 
 	client, err := g.httpClientWithCreds()
 	if err != nil {
-		logrus.Warnf("Problem creating http client to check git remote sha of repo [%v]: %v", g.URL, err)
+		g.log.Error(err, "Problem creating http client to check git remote sha of repo", "url")
```

Comment:
Do we need an additional `"url"` parameter there, given that it's already provided in `g.log`? Plus, the signature of `Error` suggests that we should provide key/value _pairs_. Or am I missing something?
```suggestion
		g.log.Error(err, "Problem creating http client to check git remote sha of repo")
```